packages/api/src/util/errorCodes.js

```diff
@@ -146,6 +166,33 @@ export async function handleObjectionError(err, res, trx) {
       type: 'UnknownDatabaseError',
       data: {},
     });
+  }
+}
+
+// Objection response
+const isNotIterable = 'is not iterable';
+// Plain knex response for tests
+const isNotIterableTests =
+  'Invalid attempt to iterate non-iterable instance.\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method.';
```

Comment:
I'm not a fan of this parsing of specific error messages tbh, it seems brittle -- if they change the message in the slightest this will break. Out of all the options, I think I'd rather just keep the 500 or keep the explicit checks for 400s. Thoughts?

Reply:
Last approach it is! lol 

@antsgar should I make a tech debt ticket (spike, rfc) about specifying our backend request shape. There has been a few suggestions in the past about improving our backend in this regard and we have OAS stuff but to my knowledge we haven't chosen a direction for this improvement yet.